dsa-solutions/lc-solutions/0200-0299/0230-Kth-smallest-element-in-BST.md

```diff
@@ -0,0 +1,78 @@
+---
+id: kth smallest element in binary search tree
```

Comment:
replace `kth smallest element in binary search tree` to `kth-smallest-element-in-binary-search-tree`